filter.txt

```diff
@@ -1378,6 +1378,7 @@ z9-we.com#@#a[href*="//z9-we.com"]
 ||dlv.hooniad.com^
 ||dnhfi5nn2dt67.cloudfront.net/hls.min.js^
 ||docdocdoc.co.kr/banner/rollbanner_*.html^$domain=docdocdoc.co.kr
+||dobaklife1.com/data/apms/
```

Comment:
please check my commit - banner images also may be blocked. They are large.
https://github.com/AdguardTeam/AdguardFilters/commit/549f79c1779238ea3ae3468f79f489a4c4a2a60c